guides/V1_share_space_template.md

```diff
@@ -57,3 +57,16 @@ mutation {
   }
 }
 ~~~~
+
+## Workspace Share URL
+
+Once shared, templates can be accessed in Workspace via a share URL.
+
+The format of the workspace URL is
```

Comment:
Maybe `"To view the shared template, go to:"`? Perhaps a bit more clarification about what it means to "view" - what would you see?